_assets/styles/scss/05-regions/_team-member-list.scss

```diff
@@ -21,9 +21,9 @@ plugin to create a `<picture>` element for the team member photo.
     <h2 class="heading--underline--magenta">Team</h2>
     <div class="region--team-member-list__list">
       <div class="team-member-bio">
-        <a class="team-member-bio__link" href="/company/anne-tomasevich/">
+        <a class="team-member-bio__link" href="/company/">
```

Comment:
It would be more ideal to replace this with a different team member but not worth changing, just a quick note.